taxi/forms.py

```diff
@@ -0,0 +1,49 @@
+from django.forms import ModelForm, ValidationError, CheckboxSelectMultiple
+from .models import Car, Driver
+
+
+class DriverLicenseUpdateForm(ModelForm):
+    LICENSE_LEN = 8
+    INT_START_INDEX = 3
+
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+        if len(license_number) != 8:
+            raise ValidationError(
+                f"Consist only of {DriverLicenseUpdateForm.LICENSE_LEN}"
+                "characters"
+            )
+        if (
+            not license_number[
+                : DriverLicenseUpdateForm.INT_START_INDEX
```

Comment:
```suggestion
                : self.INT_START_INDEX
```
You can use self. Fix in all places